Branch-TestBed/Branch-SDK Debugger Tests/Branch_SDK_Debugger_Tests.m

```diff
@@ -0,0 +1,65 @@
+//
+//  Branch_SDK_Debugger_Tests.m
+//  Branch-TestBed
+//
+//  Created by Qinwei Gong on 2/23/15.
+//  Copyright (c) 2015 Branch Metrics. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+#import "BNCPreferenceHelper.h"
+#import "BranchServerInterface.h"
+#import "Nocilla.h"
+
+@interface Branch_SDK_Debugger_Tests : XCTestCase {
+    
+@private
+    BNCPreferenceHelper *prefHelper;
+    BranchServerInterface *serverInterface;
+}
+
+@end
+
+@implementation Branch_SDK_Debugger_Tests
+
+- (void)setUp {
+    [super setUp];
+    
+    [[LSNocilla sharedInstance] start];
```

Comment:
Note that you're starting and stopping Nocilla between each test. In the Github example, it suggests doing so at the start and stop of the suite (the `+ (void)setup` instead of `- (void)setup`.